rosbag2_examples/rosbag2_examples_py/rosbag2_examples_py/simple_bag_reader.py

```diff
@@ -41,6 +41,7 @@ def timer_callback(self):
                 continue
             self.publisher.publish(msg[1])
             self.get_logger().info('Publish serialized data to ' + msg[0])
+            break
```

Comment:
I believe that what needs to be done here is to break out the while loop, so that we can publish the data only for `chatter` topic for every iteration.